escoba/tests/e2e/custom-assertions/elementCount.js

```diff
@@ -0,0 +1,19 @@
+// A custom Nightwatch assertion.
```

Comment:
what is this doing?

Reply:
this is also vue cli boilerplate.  figured we'd implement tests eventually